bundles/binding/org.openhab.binding.rfxcom.test/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComMessagesTest.java

```diff
@@ -33,7 +29,7 @@ public void checkNotImplementedPackets() throws Exception {
                 }
             }
             if (errors.length() > 0) {
-                throw new Exception("Packet messages not implemented : " + errors);
+                System.out.println("Packet messages not implemented : " + errors);
```

Comment:
In the original test an exception was thrown, which caused tests always to fail since not all RFXCom protocols are implemented and most likely never (or not in the near future) will be. The tests were also not included in the parent pom.

So I changed the behaviour to at least do a print out of any unsupported protocol, but not fail the test.

 I could change it to logging but I don't know whether a logging framework is already imported there. Any suggestions?